search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -33,6 +33,8 @@
    :entry-id :string
    :entry-title :string
    :exclude :exclude
+   :granule-data-format :string
+   :granule-data-format-h :string
```

Comment:
I think this should be `:granule-data-format-h :humanizer`. I am worried about your test now. How did they pass?